pymoo/core/individual.py

```diff
@@ -63,36 +65,36 @@ def __init__(
             in the ``Individual``.
         """
         # set decision variable vector to None
-        self._X = None
+        self._X: np.ndarray | None = None
```

Comment:
Is it necessary to assign `None` to these, or could they be `empty` as in `reset` below?